hw3/01-routing.js

```diff
@@ -1,31 +1,28 @@
-const http = require('http');
-const port = process.env.PORT || 5000;
+const http = require(`http`);
```

Comment:
You might want to merge your old PR (3rd excercise PR) as I can see these changes in this PR as well.

Reply:
@juhiaugustine  Oh right! I was waiting to be reviewed before merging hw3, sorry for the trouble.